frontend/src/metabase/actions/components/ActionViz/ActionDashcardSettings.styled.tsx

```diff
@@ -67,10 +67,7 @@ export const ParameterMapperContainer = styled.div`
   flex-direction: column;
   overflow-y: auto;
   flex: 1;
-  padding-top: ${space(1)};
```

Comment:
I can't say I like this particular change TBH.
Sometimes separate properties convey the meaning better than a shorthand one.